test/e2e/framework/framework.go

```diff
@@ -54,11 +54,9 @@ import (
 )
 
 const (
-	maxKubectlExecRetries = 5
-	// DefaultNamespaceDeletionTimeout is timeout duration for waiting for a namespace deletion.
-	// TODO(mikedanese): reset this to 5 minutes once #47135 is resolved.
-	// ref https://github.com/kubernetes/kubernetes/issues/47135
-	DefaultNamespaceDeletionTimeout = 10 * time.Minute
+	maxKubectlExecRetries           = 5
+    // DefaultNamespaceDeletionTimeout is timeout duration for waiting for a namespace deletion.
+	DefaultNamespaceDeletionTimeout = 5 * time.Minute
```

Comment:
@smarterclayton do we still need this? I thought you removed the namespace deletion wait altogether

Reply:
that's https://github.com/kubernetes/kubernetes/pull/82439 , right?